engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/StorageOrchestrator.java

```diff
@@ -215,7 +219,20 @@ public MigrationResponse migrateData(Long srcDataStoreId, List<Long> destDatasto
         return handleResponse(futures, migrationPolicy, message, success);
     }
 
-    protected Pair<String, Boolean> migrateCompleted(Long destDatastoreId, DataStore srcDatastore, List<DataObject> files, MigrationPolicy migrationPolicy) {
+    private String getObjectType(DataObject dataObject) {
+        if (dataObject instanceof VolumeInfo) {
+            return "volume";
```

Comment:
Can return string from enum types here:
https://github.com/apache/cloudstack/blob/51d4e5475f3a461b7f1aef39aded5e09773759dd/api/src/main/java/com/cloud/agent/api/to/DataObjectType.java#L22

Reply:
wouldn't dataObject.getType().getName() make more sense? and do away with the stacked if-statements completely, @sureshanaparti and @Pearl1594 ?

Reply:
Yes @DaanHoogland  - agree

Reply:
> wouldn't dataObject.getType().getName() make more sense? and do away with the stacked if-statements completely, @sureshanaparti and @Pearl1594 ?

+1 @DaanHoogland